kythe/go/extractors/config/wrapper/gradle_build_modifier.go

```diff
@@ -20,11 +20,11 @@ allprojects {
 `
 
 // This matches a line which sets the javac to use Kythe's javac-wrapper.sh
-var kytheMatcher = regexp.MustCompile(`^\s*options\.forkOptions\.executable\ =\ '/opt/kythe/extractors/javac-wrapper.sh'\s*$`)
+var kytheMatcher = regexp.MustCompile(`\s+options\.forkOptions\.executable\ =\ '/opt/kythe/extractors/javac-wrapper.sh'`)
```

Comment:
_Must_ there be spaces before `options`, or is this simply trying to ensure a word break? 

If it's the latter: Would `\b` suffice?

Reply:
I guess if we want to be precise about it, actually I should stuff a \n out front there, because we really want this to be a separate new statement with no preceding // comment markers or anything.

Then \s+ to match arbitrary whitespace